File: 8.js
// Sigma n^n/n! (Recursion)
function Factorial(start, end) {
    if (start <= end) {
        return start * Factorial(start + 1, end);
    }
    return 1;
}

function Summation(start, end) {
    if (start <= end) {
        return start ** start / Factorial(1, start) + Summation(start + 1, end);
    }
    return 0;
}

console.log(Summation(1, 10));

File: 8.c
// Sigma n^n/n! (Recursion)
#include <stdio.h>
#include <math.h>
int Factoral(int start, int end);
int Summation(int start, int end);
int main()
{
    printf("%d\n", Summation(1, 10));
    return 0;
}
int Factoral(int start, int end)
{
    if (start <= end)
    {
        return start * Factoral(start + 1, end);
    }
    return 1;
}
int Summation(int start, int end)
{
    if (start <= end)
    {
        return pow(start, start) / Factoral(1, start) + Summation(start + 1, end);
    }
    return 0;
}